docs/src/main/asciidoc/_admin-guide/Domain_Setup.adoc

```diff
@@ -130,30 +118,34 @@ will be used, one of which will be set on most operating systems. If
 neither is set the name will be the value of
 `InetAddress.getLocalHost().getHostName()`.
 
-A security realm needs to be defined to hold the identity of the slave.
-Since it is performing a specific purpose I would suggest a new realm is
-defined although it is possible to combine this with an existing realm.
+An `authentication-context` needs to be defined in the `elytron` subsystem
+to contain the identity of the host controller.
 
 [source,xml,options="nowrap"]
 ----
-<security-realm name="SlaveRealm">
-  <server-identities>
-    <secret value="cE3EBEkE=" />
-  </server-identities>
-</security-realm>
+<subsystem xmlns="urn:wildfly:elytron:14.0" final-providers="combined-providers" disallowed-providers="OracleUcrypto">
+    <authentication-client>
+        <authentication-configuration sasl-mechanism-selector="DIGEST-MD5"
+                                      name="slaveHostAConfiguration"
```

Comment:
There are couple of places like here where more instances of problematic language is introduced. Since these are not hard coded values, can you follow the guideline and replace these? 

https://github.com/conscious-lang/conscious-lang-docs/blob/main/faq.md

Reply:
At the moment the relevant configuration files remain as host-master.xml and host-slave.xml we really need some new names for these files.

Reply:
Right, I assume those have a separate Jira as it's beyond just documentation. What I am referring to here is specifically e.g. context names which are arbitrary.